ctapipe/image/muon/muon_integrator.py

```diff
@@ -68,7 +68,8 @@ def __init__(self, mirror_radius, hole_radius, pixel_width=0.2,
         self.prediction = 0
         self.minlambda = 300.e-9
         self.maxlambda = 600.e-9
-        self.photemit = alpha * (self.minlambda**-1 - self.maxlambda**-1)  # 12165.45
+        self.photemit = 1.0 / 137.0 * (self.minlambda**-1 -
```

Comment:
Why replace a well-known constant with two magic numbers?

Reply:
at least use `scipy.constants.alpha` (or `scipy.constants.inverse_alpha`)

Reply:
or `astropy.constants.alpha` which is also there, and has references to where the value comes from automatically